From 3a4bceeffae5b170af523cd9f2798d5a0d14a2d7 Mon Sep 17 00:00:00 2001
From: Darren Hart <dvhart@linux.intel.com>
Date: Tue, 6 Oct 2015 23:17:04 +0100
Subject: [PATCH] MAINTAINERS: Add drivers/platform/olpc to
 drivers/platform/x86

Andy Shevchenko sent me a simple patch that's been sitting on LKML for
10 months for OLPC and asked if I'd take it via platform/drivers/x86.
OLPC appears to be unmaintained per MAINTAINERS. Add it to
platform/drivers/x86 (unless someone else really wants it!)

Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index adb8e58..37997b3 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -11459,6 +11459,7 @@ L:	platform-driver-x86@vger.kernel.org
 T:	git git://git.infradead.org/users/dvhart/linux-platform-drivers-x86.git
 S:	Maintained
 F:	drivers/platform/x86/
+F:	drivers/platform/olpc/
 
 X86 MCE INFRASTRUCTURE
 M:	Tony Luck <tony.luck@intel.com>
-- 
1.9.3

